spec/abci/abci++_app_requirements.md

```diff
@@ -1025,16 +1051,19 @@ First of all, switching to a version of CometBFT with vote extensions, requires
 For a detailed description on the upgrade path, please refer to the corresponding 
 [section](./../../docs/rfc/rfc-100-abci-vote-extension-propag.md#upgrade-path) in RFC-100.
 
-There is a newly introduced [**new consensus parameter**](./abci%2B%2B_app_requirements.md#abciparamsvoteextensionsenableheight): `VoteExtensionsEnableHeight`. 
+There is a newly introduced [**consensus parameter**](./abci%2B%2B_app_requirements.md#abciparamsvoteextensionsenableheight): `VoteExtensionsEnableHeight`. 
 This parameter represents the height at which vote extensions are 
 required for consensus to proceed, with 0 being the default value (no vote extensions).
-
+A chain can enable vote extensions either:
+* at genesis by setting `VoteExtensionsEnableHeight` to be equal to the `InitialHeight`
```

Comment:
```suggestion
* at genesis by setting `VoteExtensionsEnableHeight` to be equal, e.g., to the `InitialHeight`
```
I'm suggesting this to tackle @adizere's concern, I think he's got a point (shouldn't imply that, if you set it at genesis then you MUST set it to _initialHeight_)